functionary/train/train_lora.py

```diff
@@ -47,7 +42,11 @@ class ModelArguments:
 @dataclass
 class DataArguments:
     train_data_path: str = field(default=None, metadata={"help": "Path to the training data."})
+    training_ratio: float = field(default=1.0, metadata={"help": "percentage of data used for training"})
```

Comment:
This might be confusing as we already splitted the train and eval before the training. If your goal is to test on a small dataset, you can pass "--max_steps" (https://huggingface.co/docs/transformers/main_classes/trainer#transformers.TrainingArguments.max_steps)

Reply:
Oh, the purpose is not training on a small number of steps, but to **reduce the time for tokenizing**. When we want to test the training, we want to run on small dataset first so we don't want to **wait for 6-7 minutes for tokenizing**